test/CMakeLists.txt

```diff
@@ -179,13 +197,24 @@ macro(try_add_performance_test)
     ${rmw_implementation}
     "ROS2"
     ${rmw_implementation}
+    "async"
```

Comment:
Won't this be completely wrong for `rmw_cyclonedds_cpp`?

Reply:
Yes, that's right, cyclonedds only allows to use sync mode. I can add some more logic but the parameters used internally are not affecting the behaviour of cyclonedds

Reply:
So...has this been addressed? I thought part of the point of these sync/async changes was to ensure that we're comparing the results fairly. If we specifically present the CycloneDDS results as asynchronous even though they're synchronous, it would be even worse.